company/goals/index.md

```diff
@@ -0,0 +1,72 @@
+# Goals
+
+Our company goals are public:
+
+- [New *continuously updated* goals starting as of FY20-Q3 (upcoming, not publicly visible yet, will be moved to handbook soon)](https://docs.google.com/document/d/1Z6avFUmnGW-ZC6zrktqqQd_g5mhBE96C_mTwXwFc1H4/edit#)
+- Historical snapshots:
+  - [FY20-Q2 (active)](2020_q2.md)
+  - [FY20-Q1](2020_q1.md)
+  - [CY19-Q4](2019_q4.md)
+  - [CY19-Q3](2019_q3.md)
+
+## Philosophy
+
+We believe:
+
+- "Plans are useless, but planning is essential."
+- "No plan survives first contact with the customer."
+- "A plan is only useful for determining the immediate next thing to do. Beyond that, you must be ready to abandon the plan in the face of new evidence."
+- Discarding a goal that turns out to be wrong is better than following through on the wrong goal. People know this intellectually but disregard it in practice.
+- Each problem we're solving has a different (and often unknown/unknowable) timeline and rate of progress.
+- Only people with skin in the game can set useful goals.
+
+## Goals are continuous, not quarterly
+
+1. Goals are continuously updated, and each goal specifies its own time period. They are not (for example) quarterly.
+   - No single update/review cadence (such as quarterly) makes sense for all goals.
+1. Change your goal immediately if/when you discover it's the wrong goal.
+   - Reflect on why you needed to change the goal (so you can learn), but don't keep working toward the wrong goal.
+
+## Choosing goals
+
+A goal's format is:
+
+> **Goal title** \
+> Description of the goal and how to evaluate whether we met the goal (with a link to an automatically updated metrics dashboard, if appropriate).
```

Comment:
In support of the first item below, should the person responsible be listed by the goal? 

```suggestion
> **Goal title** \
> Owner: the name of the single person who is responsible. 
> Description of the goal and how to evaluate whether we met the goal (with a link to an automatically updated metrics dashboard, if appropriate).
```

Reply:
I'm not sure. In most cases, all of the goals in a given list of goals will have the same owner (I think). So it might be clearer to state that in the section header. Let's see how it goes when we start to compile the goals.